packages/flutter_hooks/lib/src/async.dart

```diff
@@ -117,9 +117,9 @@ class _FutureStateHook<T> extends HookState<AsyncSnapshot<T>, _FutureHook<T>> {
   Object? get debugValue => _snapshot;
 }
 
-/// Subscribes to a [Stream] and return its current state in an [AsyncSnapshot].
+/// Subscribes to a [Stream] and returns its current state as an [AsyncSnapshot].
 ///
-/// * [preserveState] defines if the current value should be preserved when changing
+/// * [preserveState] determines if the current value should be preserved when changing
 /// the [Future] instance.
```

Comment:
I think this is supposed to be `Stream`?
```suggestion
/// the [Stream] instance.
```